micrometer/src/main/scala/com/avast/sst/micrometer/interop/Http4sServerMetrics.scala

```diff
@@ -0,0 +1,45 @@
+package com.avast.sst.micrometer.interop
+
+import java.util.concurrent.TimeUnit
+
+import cats.effect.Sync
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import com.avast.sst.micrometer.HttpStatusMetrics
+import io.micrometer.core.instrument.MeterRegistry
+import org.http4s.metrics.{MetricsOps, TerminationType}
+import org.http4s.{Method, Status}
+
+import scala.language.higherKinds
+
+class Http4sServerMetrics[F[_]: Sync](meterRegistry: MeterRegistry) extends MetricsOps[F] {
```

Comment:
this class contains implementation details which are incidental, and users shouldn't need to care about
the class can be made private and copanion object can have method `make` which will return just `MetricsOps`

Reply:
Right.